src/digitalIdentity-frontend/src/app/services/backend-http-service/backend-http-service.service.ts

```diff
@@ -29,7 +29,7 @@ export class BackendHttpService {
           params: params,
         })
         .pipe(
-          timeout(20000),
+          timeout(30000),
           // timeout
```

Comment:
The timeout could be even longer than 30000 (try 50000 to be sure that it works)